examples/multimap/batch_operations/main.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+type employee struct {
+	Name       string
+	Department string
+}
+
+var (
+	alice = employee{
+		Name:       "Alice",
+		Department: "Engineering",
+	}
+	bob = employee{
+		Name:       "Bob",
+		Department: "Engineering",
+	}
+	john = employee{
+		Name:       "John",
+		Department: "Accounting",
+	}
+)
```

Comment:
Would it be better to move this to its own function, like `createEmployees`?

Reply:
Refactored samples for simplicity as discussed. Resolving this comment, since it is no longer applicable.